packages/fx-core/resource/yaml.schema.json

```diff
@@ -1133,6 +1134,54 @@
         }
       }
     },
+    "fileUpdateLaunchUrlInLaunchSettings": {
```

Comment:
Make the action general to create a new profile or update an existing one, e.g.,
``` yaml
- uses: file/createOrUpdateDebugProfile
  with:
    name: Microsoft Teams (browser) # the name of launch profile
    appId: ${{TEAMS_APP_ID}}
    loginHint: true # can be optional
    host: teams # can be optional
    ...
```

If there's no such a profile, generate a new one.
If there's already such a profile, update its `launchUrl`.

Reply:
Already update local yml file.